playwright/features/pub_yearFilter.feature

```diff
@@ -8,12 +8,3 @@ Feature: Publication Year Filter
         And I fill in the "publication year to filter" with "1900"
         And I click the "publication year apply button"
         Then the "first search result edition" should be displayed
-
-    Scenario: As a user I filter by publication years and verify the search results
-        Given I go to the "home" page
-        When I fill in the "homepage search box" with "Jane Austen"
-        And I click the "search button"
-        And I fill in the "publication year from filter" with "1800"
-        And I fill in the "publication year to filter" with "1835"
-        And I click the "publication year apply button"
-        Then only editions published between year "1800" and "1835" should be displayed
```

Comment:
Why is this test being removed? Seems like the scenario itself is still worth testing?

Reply:
There's a scenario for that under publication.feature @samanthaandrews